code/modules/jobs/job_types/emt.dm

```diff
@@ -0,0 +1,41 @@
+/datum/job/emt
+	title = "Paramedic"
+	flag = EMT
+	department_head = list("Chief Medical Officer")
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 5
+	spawn_positions = 3
+	supervisors = "the chief medical officer"
+	selection_color = "#ffeef0"
+
+	outfit = /datum/outfit/job/emt
+
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_VIROLOGY, ACCESS_MECH_MEDICAL, ACCESS_RESEARCH, ACCESS_ENGINE, ACCESS_MINERAL_STOREROOM, ACCESS_MAINT_TUNNELS)
```

Comment:
🤔

Reply:
again, placeholder, need suggestions, it's a WIP

Reply:
They literally have more access than actual doctors.

Reply:
They should have _less_ access than doctors plus maint.

Reply:
haven't done it yet, was going to remove surgery. also that's the access for when there is a skeleton crew, you're looking for minaccess

Reply:
actually, this does work, albeit the objective is a bit weird for EMTs

Reply:
I know. I think both lists are excessive.